### PR TITLE
Create Project management dropdown

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

- Splits ClusterSelection component from PageHeader
- Adds Project Management Menu

## Fixes

- Show cluster selection as the default
- Check Kubernetes config path validity correctly
- Fix hide Cluster Selection action

## How to test it

-

## Screenshots

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/PageHeader/ProjectSelection.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      return DateTime.fromISO(isoDate).toRelative();
+    }
+    return '';
+  };
+
+  const projectMenu = () => {
+    return (
+      <StyledProjectMenu>
+        <StyledProjectsMenuContainer>
+          <StyledSearch placeholder="Search" value={searchText} onChange={handleProjectSearch} />
```
Comment (olensmar) [inline src/components/organisms/PageHeader/ProjectSelection.tsx:107]: We should add tooltips to these 4:
- Search: "Search for project by name"
- Browse: "New project from existing folder"
- Empty project: "New Empty Project"
- Project from template: "New Project from Template"

File: src/components/organisms/PageHeader/ProjectSelection.tsx [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      isClusterSelectorVisible={isClusterSelectorVisible}
+      overlay={projectMenu}
+      placement="bottomCenter"
+      arrow
+      trigger={['click']}
+      onVisibleChange={setIsDropdownMenuVisible}
+      visible={isDropdownMenuVisible}
+    >
+      <StyledProjectButton>
+        <StyledFolderOpenOutlined />
```
Comment (olensmar) [inline src/components/organisms/PageHeader/ProjectSelection.tsx:220]: Tooltip needed here also: "Select and manage your projects" ?

**Comment by olensmar** [comment]
@tommywhitehead see screenshot below - do we really need this marker to the left? We don't have it anywhere else when hovering objects (navigators, etc)

image

**Comment by erdkse** [comment]
@olensmar If everything is ok except design - let's merge this and fix design-related issues in a separated PR.

File: src/components/organisms/PageHeader/Styled.tsx [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+import Col from '@components/atoms/Col';
+import Header from '@components/atoms/Header';
+import Row from '@components/atoms/Row';
+
+import {AppBorders} from '@styles/Borders';
+import Colors, {BackgroundColors, FontColors} from '@styles/Colors';
+
+const {Search} = Input;
+
+export const StyledButton = styled(Button)`
```
Comment (devcatalin) [inline src/components/organisms/PageHeader/Styled.tsx:29]: just to be consistent with other styled components from the code base, could you remove the "Styled" prefix from the components in this file?
then you would import them like this: `import * as S from "./Styled"` and use the components like this:
`<S.Button />` , `<S.ClusterButton />` or `<S.ClusterActionButton />` etc

note: if you want to export a `Button` component from this but it has the same name with the Antd `Button` , then you can import the button from antd like this:
`import { Button as RawButton } from "antd"`

**Comment by devcatalin** [comment]
Also please add the `GlobalScrollbarStyle` to the scrollbar here: 
image

**Review by olensmar — APPROVED** [review]
I have some styling/wording "reservations" but we can discuss them with @tommywhitehead and fix in later PRs

**Comment by erdkse** [comment]
@devcatalin Could you test tables scrollbar style? I added it but not sure if it is applied.

**Pull request merged**